db/migrate/20210622184700_rename_flavors_cpus_to_match_hardware.rb

```diff
@@ -0,0 +1,7 @@
+class RenameFlavorsCpusToMatchHardware < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :flavors, :cpus, :cpu_total_cores
+    rename_column :flavors, :cpu_cores, :cpu_cores_per_socket
+    add_column    :flavors, :cpu_sockets, :integer, :default => 1
```

Comment:
NOTE since this is commonly used as a denominator and also not commonly set, default to 1 (hardwares.cpu_sockets does the same thing)